plugin/trino-hive/src/main/java/io/trino/plugin/hive/orc/OrcDeletedRows.java

```diff
@@ -62,6 +65,8 @@
     private static final int BUCKET_ID_INDEX = 1;
     private static final int ROW_ID_INDEX = 2;
 
+    private static final long DELETED_ROWS_SYSTEM_MEMORY_INCREASE_YIELD_THREHOLD = 32 * 1204 * 1024;
```

Comment:
we can maybe yield every # of rows processed (eg 100_000)?
this would have other advantages, like query/work interruptability

Reply:
Hmm - isn't a single row always of fixed size for deleted row ids? Then the value in MBs directly translates to a specific number of rows. 
And given the fact that mechanism is built for better memory accounting, having threshold defined in MBs is more readable to me.

Reply:
yielding has other advantages than memory accounting (query/work interruptability)
btw we could weave usage of `io.trino.operator.DriverContext#getYieldSignal` here.

ok to have this as a TODO for know; then memory-related accounting has more sense.